test/integration/scripting_spec.mjs

```diff
@@ -2395,7 +2395,10 @@ describe("Interaction", () => {
           const inputSelector = getSelector("33R");
           await page.click(inputSelector);
           await page.type(inputSelector, "7");
+          await waitForSandboxTrip(page);
```

Comment:
nit: just use `typeAndWaitForSandbox` and probably everywhere we've a `page.type` in this file.

Reply:
Good idea! This way we can simplify the test even further, and I've verified that the new code still works with the delay introduced.